helm/kong-app/templates/psp.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ template "kong.serviceAccountName" . }}-psp
+  labels:
+    app: {{ template "kong.name" . }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+spec:
+  privileged: false
```

Comment:
@glitchcrab Doesn't this need to be true so it can run as root?

AIUI the default PSP has `privileged: false` so this should override it here just for Kong.

Reply:
@rossf7 yes it did - I corrected that but still no joy :(

Reply:
Actually just looking at this (i've been through many permutations), privileged allows access to devices on the host which isn't necessary in this case.